packages/@lwc/engine-dom/src/renderer-factory.ts

```diff
@@ -31,12 +31,15 @@ export type RendererAPIType<Type> = Type extends RendererAPI ? RendererAPI : San
  * A factory function that produces a renderer.
  * Renderer encapsulates operations that are required to render an LWC component into the underlying
  * runtime environment. In the case of @lwc/enigne-dom, it is meant to be used in a DOM environment.
- * Example usage:
+ * @param baseRenderer Either null or the base renderer imported from 'lwc'.
+ * @returns The created renderer
+ * @example
  * import { renderer, rendererFactory } from 'lwc';
  * const customRenderer = rendererFactory(renderer);
- * @param baseRenderer Either null or the base renderer imported from 'lwc'.
  */
 export function rendererFactory<T extends RendererAPI | null>(baseRenderer: T): RendererAPIType<T> {
+    // `process.env.RENDERER` is replaced by rollup with an object, not a string. Gross!
```

Comment:
```suggestion
    // `process.env.RENDERER` is replaced by rollup with an object, not a string
```

I prefer not to editorialize here. In the absence of a better solution, "gross" doesn't really explain what the issue is or what we could do better. 🙂